### ISSUE TITLE
Poetry build produces incompatible wheel

### ISSUE THREAD

**Issue by severinsimmler** [python-poetry/poetry#3659]
- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: Ubuntu 18.04.5 LTS
- **Poetry version**: 1.1.4
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: [pyproject.toml](https://github.com/severinsimmler/chaine/blob/master/pyproject.toml)

## Issue
I'm building and deploying wheels on GitHub Actions. Since [this build](https://github.com/severinsimmler/chaine/runs/1853508242?check_suite_focus=true) (check out the "Build wheels" stage), `poetry build --format wheel` produces a wheel `manylinux_2_27_x86_64.whl`, but it used to be `manylinux2014_x86_64.whl`. I couldn't find any documentation on the `manylinux_2_27_x86_64.whl` pattern. For reference, [this build](https://github.com/severinsimmler/chaine/runs/1682141114?check_suite_focus=true) produced the `manylinux2014_x86_64.whl` wheels.

However, this leads in _another_ project with this wheel as a dependency to a runtime error when I run `poetry update`:

```
  • Updating chaine (1.1.0 -> 1.1.2): Failed

  RuntimeError

  Unable to find installation candidates for chaine (1.1.2)
```

Note that 1.1.0 is the `manylinux2014_x86_64.whl`.

Upgrading with `pip` works just fine:

```sh
$ pip install --upgrade chaine
Requirement already satisfied: chaine in ./.pyenv/versions/3.9.1/lib/python3.9/site-packages (1.1.0)
Collecting chaine
  Using cached chaine-1.1.2-cp39-cp39-manylinux_2_27_x86_64.whl (902 kB)
Installing collected packages: chaine
  Attempting uninstall: chaine
    Found existing installation: chaine 1.1.0
    Uninstalling chaine-1.1.0:
      Successfully uninstalled chaine-1.1.0
Successfully installed chaine-1.1.2
```

**Comment by severinsimmler** [comment]
This issue might be a duplicate of https://github.com/python-poetry/poetry/issues/3584

**Comment by severinsimmler** [comment]
Especially:

> Alright, so, `poetry.utils.env.supported_tags` is an alias for `packaging.tags.sys_tags`. But for some reason the list of supported tags are drastically lower than what should be available.  From `manylinux_2_5_x86_64` to `manylinux_2_28_x86_64` are somehow missing.
> 
>I am looking at this file: https://github.com/python-poetry/poetry/blob/3f6220dedb6d824e14e0c703485bbf76c32af3db/poetry/utils/env.py

_Originally posted by @xrogaan in https://github.com/python-poetry/poetry/issues/3584#issuecomment-761910149_


And also my project makes use of glibc:

>If poetry is installed through the official installer all dependencies are vendored (`$HOME/.poetry/lib/poetry/_vendor`). The version of `packaging` shipped is 20.4, which doesn't support glibc tags.
>
>See:
>* https://github.com/pypa/packaging/compare/20.4...20.8
>* https://github.com/pypa/packaging/issues/280

_Originally posted by @xrogaan in https://github.com/python-poetry/poetry/issues/3584#issuecomment-762342725_

**Comment by finswimmer** [comment]
Closing this as a duplicate of #3584
